### PR TITLE
Initialize using stored networkId

### PR DESCRIPTION
**Pull request by CedrikNikita**

### CODE REVIEW

**Comment by Liubov-crypto** [comment]
Tested, for me, the error is still reproducible.

File: .env [hunk 1]
```diff
@@ -1,4 +1,4 @@
-VUE_APP_DEFAULT_NETWORK=ae_mainnet
+VUE_APP_DEFAULT_NETWORK=ae_uat
```
[the remark this answers is not in the document]
Comment (CedrikNikita) [reply .env:1]: That is for develop version only, the production is still mainnet.

**Comment by Liubov-crypto** [comment]
I am still getting this issue:

https://user-images.githubusercontent.com/69896204/182647329-c3e9e9da-9dcb-4067-b30b-032935e07a1c.mov

wallet: commit 4fc9539 / v.0.9.10, 
also I tested with develop version of wallet. 
cons1
cons dex

**Review by kenodressel — APPROVED** [review]
Yes its not a fix for #434. But regardless it should initalize the dex faster.

**Comment by CedrikNikita** [comment]
> I am still getting this issue:

First of all you need to enable the popups in the browser for that paricular page, and it should working fine even without "Open my wallet" option. But even if you do click "Open my wallet" it should work fine if you would not block cookies in the incognito mode, or just no block cookies.

**Comment by Liubov-crypto** [comment]
> https://pr-440-dex-ui.stg.aepps.com

Tested, LGTM. 

> > I am still getting this issue:
> 
> First of all you need to enable the popups in the browser for that paricular page, and it should work fine even without "Open my wallet" option. But even if you do click "Open my wallet" it should work fine if you would not block cookies in the incognito mode, or just not block cookies.
> 
> > Yes its not a fix for #434. But regardless it should initalize the dex faster.
> 
> It is not a fix since everything is working already. This PR is just saving us from jumping from Mainnet to Testnet, if the Testnet is chosen in the wallet.

Tested, LGTM.

**Pull request merged**